SmartTutor/SmartTutor.csproj

```diff
@@ -18,5 +18,6 @@
 
   <ItemGroup>
     <ProjectReference Include="..\RepositoryCompiler\RepositoryCompiler.csproj" />
+    <ProjectReference Include="..\SourceCodeTests\SourceCodeTests.csproj" />
```

Comment:
Once you move the logic and delete the project this should be discarded as well.

As a side note - the main project should never reference the test project.